pkg/daemon/ceph/client/osd.go

```diff
@@ -150,6 +154,24 @@ func OSDRemove(context *clusterd.Context, clusterName string, osdID int) (string
 	return string(buf), err
 }
 
+func OsdSafeToDestroy(context *clusterd.Context, clusterName string, osdID int) (bool, error) {
+	args := []string{"osd", "safe-to-destroy", strconv.Itoa(osdID)}
+	cmd := NewCephCommand(context, clusterName, args)
```

Comment:
It's not 100% safe since before Nautilus this command didn't have any json output... A version check would have been nice...

Reply:
Okay, Thanks, Will update

Reply:
@rohan47 Is there an issue opened to track this? thanks

Reply:
No, but I will fix it with https://github.com/rook/rook/issues/3359

Reply:
@travisn @leseb I have opened a separate issue for this https://github.com/rook/rook/issues/3421